homeassistant/components/tuya/base.py

```diff
@@ -85,6 +88,25 @@ def from_json(cls, dpcode: DPCode, data: str) -> IntegerTypeData:
             type=parsed.get("type"),
         )
 
+    def int_to_timestamp(
+        self,
+        value: float,
+        unit: str | None = None,
+    ) -> Timestamp:
+        """Convert countdown value to timestamp."""
+        now = dt_util.utcnow()
+        if value == 0:
+            return None
```

Comment:
Interestingly, this will result in an "unknown" state.

That feels not right either... However, I do not have a straight answer how to solve it otherwise.
I will get back to this, after discussing this with some other core devs.

Reply:
I don't think it's ideal either, but it seems better than displaying the current timestamp (now + 0). I haven't found if there is a way to set it to empty or idle.

Reply:
Discussion about sensor status being `unknown` for when not set:
https://github.com/home-assistant/architecture/discussions/635.